specification/ai/Azure.AI.Unified/models/models.tsp

```diff
@@ -0,0 +1,77 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../main.tsp";
+import "../common/models.tsp";
+import "@typespec/openapi";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Unified;
+
+@doc("Model Deployment Definition")
+@resource("models")
+model Model {
+  @doc("Name of the deployment for the model")
+  @visibility("read")
+  @key("deploymentName")
+  deploymentName: string;
+
+  @doc("Publisher-specific name of the model")
+  @visibility("read")
+  modelName: string;
+
+  @doc("Publisher-specific version of the model")
+  @visibility("read")
+  modelVersion: string;
+
+  @doc("Name of the model publisher")
+  @visibility("read")
+  publisher: string;
+
+  @doc("Capabilities of model")
+  @visibility("read")
+  capabilities: Record<string>;
```

Comment:
Is this to indicate if this is a chat model, embeddings model, image embeddings model, etc? If so, this should be a list of Enum values

Reply:
I opened a review comment on this in the new PR. Resolving this one.